stan/math/prim/mat/meta/vector_seq_view.hpp

```diff
@@ -0,0 +1,85 @@
+#ifndef STAN_MATH_PRIM_MAT_META_VECTOR_SEQ_VIEW_HPP
+#define STAN_MATH_PRIM_MAT_META_VECTOR_SEQ_VIEW_HPP
+
+#include <stan/math/prim/mat/fun/Eigen.hpp>
+#include <vector>
+
+namespace stan {
+  using std::vector;
```

Comment:
We don't ever want to use namespace-level includes.  It pollutes the whole namespace.  You need to spell them out in full in function arguments and returns and can use the using statements in the bodies if you use the types a lot.